### PR TITLE
Separate README from catbuffer's

### PR DESCRIPTION
**Pull request by segfaultxavi**
Now they are independent.

Related to nemtech/catbuffer#55.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,63 @@
...
 - TypeScript/JavaScript
 - Python
+
+## Requirements
+
+- Python >= 3.4
+
+## Installation
+
+1. Clone the ``catbuffer-generators`` repository:
```
Comment (fboucquez) [inline README.md:22]: Use command

`git clone --recurse-submodules https://github.com/nemtech/catbuffer-generators`

To include catbuffer when cloning

File: README.md [hunk 2]
```diff
@@ -4,11 +4,63 @@
...
+```
+
+2. Install the package requirements:
+
+```bash
+cd catbuffer-generators
+pip install -r requirements.txt
+```
+
+3. Clone the ``catbuffer`` repository inside the ``catbuffer-generators`` folder:
```
Comment (fboucquez) [inline README.md:35]: Not required when using --recurse-submodules

File: README.md [hunk 3]
```diff
@@ -4,11 +4,63 @@
...
+
+## Usage
+
+Use the scripts in the ``scripts`` folder to generate code for the different languages. For example:
+
+```bash
+scripts/generate_all.sh cpp_builder
+```
+
+Or use any of the language-specific scripts like ``generate_typescript.sh``.
```
Comment (fboucquez) [inline README.md:49]: Should we clarify that each language script also compiles and creates the artifacts? The language-specific scripts generate the artifacts in the /build folder

**Review by fboucquez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (segfaultxavi) [reply README.md:22]: Done

File: README.md (hunk 2 above)
Comment (segfaultxavi) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (segfaultxavi) [reply README.md]: Done

**Comment by segfaultxavi** [comment]
@fboucquez Any reason why this can't be landed?

**Review by fboucquez — APPROVED** [review]

**Pull request merged**